tasklist/client/src/Tasks/Task/Variables/index.tsx

```diff
@@ -54,6 +60,56 @@ type Props = {
   user: CurrentUser;
 };
 
+function createInitialFormValuesFromVariables(variables: Variable[]) {
+  const entries: Array<[string, string | null]> = [];
+  const newVariables: Array<Pick<Variable, 'name' | 'value'>> = [];
+
+  variables.forEach((variable) => {
+    if (variable.draft) {
```

Comment:
Let's invert the logic to use a explicit check like everywhere else

```suggestion
    if (variable.draft === null) {
```